nextest-runner/src/reuse_build/archiver.rs

```diff
@@ -114,16 +117,19 @@ struct Archiver<'a, W: Write> {
     builder: tar::Builder<Encoder<'static, BufWriter<W>>>,
     unix_timestamp: u64,
     added_files: HashSet<Utf8PathBuf>,
+    archive_include: Vec<ArchiveInclude>,
 }
 
 impl<'a, W: Write> Archiver<'a, W> {
+    #[allow(clippy::too_many_arguments)]
     fn new(
         binary_list: &'a BinaryList,
         cargo_metadata: &'a str,
         path_mapper: &'a PathMapper,
         format: ArchiveFormat,
         compression_level: i32,
         writer: W,
+        profile: &NextestProfile<'_, FinalConfig>,
```

Comment:
`writer` should be the last argument as it's an out param. I think `profile` should maybe be the first argument.